config/sync/field.field.node.recipe.field_r_categories.yml

```diff
@@ -12,7 +12,7 @@ entity_type: node
 bundle: recipe
 label: Categories
 description: ''
-required: false
+required: true
```

Comment:
What if the category you want doesn't exist?

Reply:
It will

Reply:
How can you guarantee that?

Reply:
Through the fact that the number of foods in the world is limited.... everything will either be chicken, beef, pork etc or vegan or vegetarian. You get me?

Reply:
Okay so your field is called categories, and my name is Garry and I'm creating a recipe. Upon seeing this field i realise that the recipe i want to create is an authentic recipe from papua nugini passed down many generations and it doesn't fit into any of those categories because the kind of food use is unusual. What then?

Reply:
You select the animal you cooking and move on, Gar.

Reply:
Are you going to include every edible species of animal in your list?

Reply:
Except for bats

Reply:
I'll wait till you're ready to take this seriously.

Reply:
Oh lighten up. It's gonna be okay. I'll have plenty of meat types and other categories like dinner or easy etc

Reply:
There's nothing worse than a developer with the hubris to assume they can foresee every possible outcome in a piece of software.

Reply:
This is me speaking as a frequent visitor of recipe sites, not a developer predicting functionality or future features

Reply:
Maybe you should consider thinking like a developer.

Reply:
The point of my comment was that this is not a programming issue in my opinion. I made the change, but make no mistake that I am thinking as a developer.

Reply:
You're not because the solution (if there is one) isn't to make the field not required.

Reply:
There is no solution for the way you are thinking. I considered it and made the field not required for now.

Reply:
Okay what if you made the field required and you allowed users to create new categories from that field If the one they want doesn't exist?

Reply:
Great idea. I'll get on that.

Reply:
It's done.